templates/repo/signature_badge.tmpl

```diff
@@ -0,0 +1,91 @@
+{{$base_classes := "show-modal tw-cursor-pointer tw-rounded-full tw-border tw-px-2 tw-py-0"}}
+{{$verified_colors := "tw-border-success-border tw-bg-success-bg tw-text-success-text"}}
+{{$unverified_colors := "tw-border-warning-border tw-bg-warning-bg tw-text-warning-text"}}
+{{if .verification.Verified}}
+	{{if eq .verification.TrustStatus "trusted"}}
+		<div class="{{$base_classes}} {{$verified_colors}}" data-modal="#{{.shortsha}}-verification-modal">Verified</div>
+	{{else if eq .verification.TrustStatus "untrusted"}}
+		<div class="{{$base_classes}} {{$unverified_colors}}" data-modal="#{{.shortsha}}-verification-modal">Unverified</div>
+	{{else}}
+		<div class="{{$base_classes}} {{$unverified_colors}}" data-modal="#{{.shortsha}}-verification-modal">Unverified</div>
+	{{end}}
+{{else}}
+	<div class="{{$base_classes}} {{$unverified_colors}}" data-modal="#{{.shortsha}}-verification-modal">Unverified</div>
+{{end}}
+<div class="ui small modal" id="{{.shortsha}}-verification-modal">
+	<div class="header tw-flex tw-items-center">
+		{{if and .verification.Verified (eq .verification.TrustStatus "trusted")}}
+			{{svg "octicon-verified" 20 "tw-mr-2"}}
+			Verified commit
+		{{else}}
+			{{svg "octicon-unverified" 20 "tw-mr-2"}}
+			Unverified commit
```

Comment:
These strings need translation. Add them to `options/locale/locale_en-US.ini` and use the key with `ctx.Locale.Tr`. Prefer to move the translation key as far up as possible inside the ini, maybe even top-level.